client/block.h

```diff
@@ -50,6 +50,8 @@ enum bi_flags {
 
 #define XDAG_BLOCK_FIELDS 16
 
+typedef char xdag_remark[32];
```

Comment:
you have no warranty that char is 8bit, you should use int8_t

Reply:
why it isn't named xdag_remark_t?

Reply:
@sgaragagghu  Add check for remarks, only ascii accept now.  Later we can add utf-8 / unicode support.